apollo-adminservice/src/main/docker/Dockerfile

```diff
@@ -4,24 +4,36 @@
 # 3. Run with: docker run -p 8090:8090 -e SPRING_DATASOURCE_URL="jdbc:mysql://fill-in-the-correct-server:3306/ApolloConfigDB?characterEncoding=utf8" -e SPRING_DATASOURCE_USERNAME=FillInCorrectUser -e SPRING_DATASOURCE_PASSWORD=FillInCorrectPassword -d -v /tmp/logs:/opt/logs --name apollo-adminservice apolloconfig/apollo-adminservice
 
 FROM openjdk:8-jre-alpine
-MAINTAINER ameizi <sxyx2008@163.com>
+LABEL maintainer="finchcn@gmail.com;ameizi<sxyx2008@163.com>"
+
+RUN echo "http://mirrors.aliyun.com/alpine/v3.8/main" > /etc/apk/repositories \
+    && echo "http://mirrors.aliyun.com/alpine/v3.8/community" >> /etc/apk/repositories \
+    && apk update upgrade \
+    && apk add --no-cache unzip \
```

Comment:
I think we need to remove the trailer `\`?
```suggestion
    && apk add --no-cache unzip
```

Reply:
fixed

Reply:
Thanks, please also help to fix the conflicts.

![image](https://user-images.githubusercontent.com/837658/108619884-cb381780-7462-11eb-8b2d-a3012b39a542.png)

Reply:
I had a little problem when trying to fix the conflicts. I refork the repository, and resoved the conflicts, but there is no push notification.
The status in tab of "File Changed" is OK.
Need I create a new PR?

Reply:
Maybe there is something wrong duing the conflict resolving process, it's ok to create a new pr.